### PR TITLE
update url name for jenkins x config

### PR DESCRIPTION
**Pull request by MarckK**
# Pull request motivation(s)
Quick update to correct site url.

### CODE REVIEW

File: configs/jenkins_x.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "index_name": "jenkins_x",
   "start_urls": [
     {
-      "url": "http://www.example.com/(?P<lang>.*?)/",
+      "url": "https://jenkins-x.io/(?P<lang>.*?)/",
```
Comment (hferentschik) [inline configs/jenkins_x.json]: 👍

**Comment by MarckK** [comment]
Thank you, @s-pace!

**Comment by s-pace** [comment]
Live

**Comment by MarckK** [comment]
Awesome! Thank you, @s-pace

**Pull request merged**